md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
```

Comment:
Interesting! 

I would add something along the lines of "These are guidelines, and not set in stone. Use your own judgement." (it allows some flexibility)